Makefile

```diff
@@ -20,5 +20,9 @@ bin/wasm-pusher:
 	mkdir -p bin/
 	cd tools/wasm-pusher && go build -o ../../bin/wasm-pusher
 
+bin/c_chaincode_example02:
```

Comment:
maybe bin/c_chaincode ? Just thinking of picking something simpler that people could easily type. Or even making another target to this that is just c_chaincode?

Reply:
renamed to c_chaincode